context/app/static/js/components/workspaces/style.js

```diff
@@ -8,4 +8,11 @@ const StyledButton = styled(WhiteBackgroundIconButton)`
   height: 36px;
 `;
 
-export { StyledButton };
+const Left = styled.div`
+  flex-shrink: 0;
+`;
+const Right = styled.div`
+  flex-shrink: 0;
+`;
+
```

Comment:
If the names are useful, I'd create a single styled component and then create `Left` and `Right` from that.